components/match2/wikis/rocketleague/match_group_input_custom.lua

```diff
@@ -207,6 +213,17 @@ function matchFunctions.getExtraData(match)
 	return match
 end
 
+function getPlayerFlag(name)
+	local data = mw.ext.LiquipediaDB.lpdb('player', {
+		conditions = '[[pagename::' .. mw.ext.TeamLiquidIntegration.resolve_redirect( name ):gsub('%s+', '_') .. ']]',
+		query = 'nationality'
+	})
+	if type(data) == 'table' and data[1] then
+		return data[1]['nationality']:lower()
```

Comment:
Generally good to check for nil (null) before doing :lower().
Is :lower() even needed here?

Reply:
The `:lower()` was left over from earlier stage when it was necessary, but it might not be necessary now since the flags module is being used rather than the template expansion

Reply:
yeah you can just kick it